src/api/mock/front-back-api-mock.json

```
@@ -0,0 +1,1762 @@
+                    "filePath": "",
+                    "databucketID": "",
+                    "sendFileAsBody": false,
+                    "rules": [
+                        {
+                            "target": "body",
+                            "modifier": "$",
+                            "value": "3yxo",
+                            "invert": false,
+                            "operator": "valid_json_schema"
+                        }
+                    ],
+                    "rulesOperator": "OR",
+                    "disableTemplating": false,
+                    "fallbackTo404": false,
+                    "default": false,
+                    "crudKey": "id",
+                    "callbacks": []
+                },
+                {
+                    "uuid": "f2a0e533-27e6-44bf-b266-f6a45ebbb5a4",
+                    "body": "{\n  \"status\": 400,\n  \"body\": {\n    \"message\": \"Bad Request\"\n  }\n}",
+                    "latency": 0,
+                    "statusCode": 400,
+                    "label": "",
+                    "headers": [],
+                    "bodyType": "INLINE",
+                    "filePath": "",
+                    "databucketID": "",
+                    "sendFileAsBody": false,
+                    "rules": [],
+                    "rulesOperator": "OR",
+                    "disableTemplating": false,
+                    "fallbackTo404": false,
+                    "default": true,
+                    "crudKey": "id",
+                    "callbacks": []
+                }
+            ],
+            "responseMode": null,
+            "streamingMode": null,
+            "streamingInterval": 0
+        },
+        {
+            "uuid": "3f457cd1-ba06-4f8b-a25c-50ec518a8576",
+            "type": "http",
+            "documentation": "",
+            "method": "get",
+            "endpoint": "problems/:problemId",
+            "responses": [
+                {
+                    "uuid": "d5a0ef36-66fc-45f8-ac9c-eba4e848be2d",
+                    "body": "{\n  \"id\": \"{{urlParam 'problemId'}}\",\n  \"title\": \"{{faker 'lorem.sentence'}}\",\n  \"authorId\": \"{{faker 'string.uuid'}}\",\n  \"isPublic\": true,\n  \"difficulty\": {{faker 'number.int' min=100 max=1000}},\n  \"statement\": \"{{faker 'lorem.sentence'}}\"\n  \"timeLimit\": 2000,\n  \"memoryLimit\": 1024,\n  \"solvedCount\": {{faker 'number.int' min=10 max=100}},\n  \"testcases\": [\n    {{#repeat 3}}\n    {\n      \"name\": \"testcase_sample\"\n      \"id\": \"{{urlParam 'testcaseId'}}\",\n      \"createdAt\": \"{{date '2020-01-01' '2024-11-05' 'yyyy-MM-dd\\'T\\'HH:mm:ss.SSS\\'Z\\''}}\",\n      \"updatedAt\": \"{{date '2024-11-06' '2024-11-11' 'yyyy-MM-dd\\'T\\'HH:mm:ss.SSS\\'Z\\''}}\"\n    }\n    {{/repeat}}\n  ]\n  \"createdAt\": \"{{date '2020-01-01' '2024-11-05' 'yyyy-MM-dd\\'T\\'HH:mm:ss.SSS\\'Z\\''}}\",\n  \"updatedAt\": \"{{date '2024-11-06' '2024-11-11' 'yyyy-MM-dd\\'T\\'HH:mm:ss.SSS\\'Z\\''}}\"\n}\n",
```

このjsonみたいな文字列、statementの行とnameの行の末尾に`,`が必要な気がします。
```suggestion
                    "body": "{\n  \"id\": \"{{urlParam 'problemId'}}\",\n  \"title\": \"{{faker 'lorem.sentence'}}\",\n  \"authorId\": \"{{faker 'string.uuid'}}\",\n  \"isPublic\": true,\n  \"difficulty\": {{faker 'number.int' min=100 max=1000}},\n  \"statement\": \"{{faker 'lorem.sentence'}}\",\n  \"timeLimit\": 2000,\n  \"memoryLimit\": 1024,\n  \"solvedCount\": {{faker 'number.int' min=10 max=100}},\n  \"testcases\": [\n    {{#repeat 3}}\n    {\n      \"name\": \"testcase_sample\",\n      \"id\": \"{{urlParam 'testcaseId'}}\",\n      \"createdAt\": \"{{date '2020-01-01' '2024-11-05' 'yyyy-MM-dd\\'T\\'HH:mm:ss.SSS\\'Z\\''}}\",\n      \"updatedAt\": \"{{date '2024-11-06' '2024-11-11' 'yyyy-MM-dd\\'T\\'HH:mm:ss.SSS\\'Z\\''}}\"\n    }\n    {{/repeat}}\n  ]\n  \"createdAt\": \"{{date '2020-01-01' '2024-11-05' 'yyyy-MM-dd\\'T\\'HH:mm:ss.SSS\\'Z\\''}}\",\n  \"updatedAt\": \"{{date '2024-11-06' '2024-11-11' 'yyyy-MM-dd\\'T\\'HH:mm:ss.SSS\\'Z\\''}}\"\n}\n",
```
適当に正規表現`\\"\\n\s*\\"`で検索してみたところ、736、802、833、864、895、926行目にも似た状態のものがあります。これはツールかなにかで作ったファイルだと思うのですが、どうしてこうなっているのでしょう...?

---

シンプルに僕が書き忘れていました(コピペを多用したので被害が広がっていた):sry:気がついてくれてありがとうございます
(ちなみにmockoonのGUIアプリを使って作成しました)